src/tables/effects.rs

```diff
@@ -8,11 +8,14 @@ use Effect::*;
 
 #[derive(Copy, Clone, Debug)]
 pub enum Effect {
-    Yes,     // no special value, just effect = yes
-    Boolean, // yes and no are both meaningful
+    Yes,
```

Comment:
And here it goes for all of them: comments after a token are for that token and should not be on the line below